doc/source/whatsnew/v3.0.0.rst

```diff
@@ -324,6 +324,7 @@ MultiIndex
 
 I/O
 ^^^
+- Bug in :class:`io.formats.excel.ExcelFormatter` when writing empty :class:`DataFrame` with :class:`MultiIndex` on both axes (:issue:`57696`)
```

Comment:
I believe `ExcelFormatter` is private; can you change to to `` :meth:`DataFrame.to_excel` ``